src/PrMerger.js

```diff
@@ -1,10 +1,10 @@
-const assert = require('assert');
-const Config = require('./Config.js');
-const Log = require('./Logger.js');
+import assert from 'assert';
+import Config from './Config.js';
+import * as Log from './Logger.js';
 const Logger = Log.Logger;
```

Comment:
Please use the following order of directives:

1. import Anubis modules, alphabetical whole-line order ("sort lines" in most editors should get this right)
2. empty line (if any Anubis modules were imported above)
3. import external modules, alphabetical whole-line order ("sort lines" in most editors should get this right)
4. empty line (if any external modules were imported above)

FWIW, Daft tries to use the same approach (but there are violations because we used a different approach earlier and because none of this is currently enforced).

Reply:
Done.